content/en/docs/concepts/storage/volume-snapshots.md

```diff
@@ -22,61 +23,62 @@ This document describes the current state of `VolumeSnapshots` in Kubernetes. Fa
 
-A `VolumeSnapshotContent` is a snapshot taken from a volume in the cluster that has been provisioned by an administrator. It is a resource in the cluster just like a PersistentVolume is a cluster resource.
+A `VolumeSnapshotContent` is a snapshot of a volume in the cluster that has been provisioned by an administrator. It is a resource in the cluster just like a `PersistentVolume` is a cluster resource.
 
-A `VolumeSnapshot` is a request for snapshot of a volume by a user. It is similar to a PersistentVolumeClaim.
+A `VolumeSnapshot` is a request for snapshot of a volume by a user. It is similar to a `PersistentVolumeClaim`.
 
-While `VolumeSnapshots` allow a user to consume abstract storage resources, cluster administrators
-need to be able to offer a variety of `VolumeSnapshotContents` without exposing
-users to the details of how those volume snapshots should be provisioned. For these needs
+Cluster administrators need to be able to offer different ways in
+which volume snapshots can be created without exposing
+users to the low level details. For these needs
 there is the `VolumeSnapshotClass` resource.
 
 Users need to be aware of the following when using this feature:
 
-* API Objects `VolumeSnapshot`, `VolumeSnapshotContent`, and `VolumeSnapshotClass` are CRDs, not part of the core API.
+* API Objects `VolumeSnapshot`, `VolumeSnapshotContent`, and `VolumeSnapshotClass` are CRDs, not part of the core API. The CSI drivers that support the volume snapshot functionality will automatically install these required CRDs.
+
 * `VolumeSnapshot` support is only available for CSI drivers.
 * As part of the deployment process, the Kubernetes team provides a sidecar helper container for the snapshot controller called `external-snapshotter`. It watches `VolumeSnapshot` objects and triggers `CreateSnapshot` and `DeleteSnapshot` operations against a CSI endpoint.
 * CSI drivers may or may not have implemented the volume snapshot functionality. The CSI drivers that have provided support for volume snapshot will likely use `external-snapshotter`.
-* The CSI drivers that support volume snapshot will automatically install CRDs defined for the volume snapshots.
+
 
 ## Lifecycle of a volume snapshot and volume snapshot content
 
-`VolumeSnapshotContents` are resources in the cluster. `VolumeSnapshots` are requests for those resources. The interaction between `VolumeSnapshotContents` and `VolumeSnapshots` follow this lifecycle:
+`VolumeSnapshotContent` objects are resources in the cluster and `VolumeSnapshot` objects are requests for those resources. The interaction between `VolumeSnapshotContent` and `VolumeSnapshot` objects follows this lifecycle:
 
 ### Provisioning Volume Snapshot
 
 There are two ways snapshots may be provisioned: statically or dynamically.
```

Comment:
replace static with pre-provisioned

Reply:
The wording here closely follows what is used for persistent volumes.

Reply:
I would like to change the wording in both places